logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -268,7 +268,7 @@ interface ConversationRepository {
         conversationID: ConversationId
     ): Either<CoreFailure, Unit>
 
-    suspend fun getConversationDetailsByMLSGroupId(mlsGroupId: GroupID): Either<CoreFailure, ConversationDetails>
+    suspend fun getConversationDetailsByMLSGroupId(mlsGroupId: GroupID): Either<CoreFailure, Conversation>
```

Comment:
suggestion: if it doesn't return `ConversationDetail` anymore then maybe we should also change the name of this function to "getConversationByMLSGroupId"?